src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
Why should the `else` case do the following, for example:

```
           if (captureOrPromotion)
               r -= r ? ONE_PLY : DEPTH_ZERO;
```

This seems awfully complicated so I want to ensure I understand the intended design.